templates/vue/src/components/modals/NodeModal/forms/ContentForm/ActivityForm.vue

```diff
@@ -140,6 +140,63 @@
                 Audio recorder
               </b-form-checkbox>
             </b-form-group>
+            <b-form-group>
+              <b-form-checkbox
+                v-model="question.answerTypes.list.enabled"
+                data-qa="question-answer-list"
+                switch
+              >
+                List Answer
+              </b-form-checkbox>
+              <div v-if="question.answerTypes.list.enabled" class="mt-2 pl-4 ml-2">
+                <div class="list-options">
+                  <div class="list-field-options">
+                    <b-form-checkbox
+                      v-model="question.answerTypes.list.maxFields.enabled"
+                      data-qa="list-max-checkbox"
+                      class="list-max-checkbox"
+                    ></b-form-checkbox>
+                    <div class="list-row-container">
+                      <div class="list-option-row">
+                        <label for="min-field">
+                          Minimum answer fields
+                        </label>
+                        <b-form-input
+                          id="min-field"
+                          v-model="question.answerTypes.list.minFields"
+                          data-qa="min-list-fields-input"
+                          placeholder="Min # of answer fields"
+                          type="number"
+                          class="list-input"
+                        ></b-form-input>
+                      </div>
+                      <div class="list-option-row">
+                        <label for="max-field">
+                          Maximum answer fields
+                        </label>
+                        <b-form-input
+                          id="max-field"
+                          v-model="question.answerTypes.list.maxFields.value"
+                          :disabled="!question.answerTypes.list.maxFields.enabled"
+                          data-qa="max-list-fields-input"
+                          placeholder="Max # of answer fields"
+                          type="number"
+                          class="list-input"
+                        ></b-form-input>
+                      </div>
+                    </div>
+                  </div>
+                  <label for="placeholder">
+                    Placeholder for answer fields (optional):
+                  </label>
+                  <b-form-input
+                    id="placeholder"
+                    v-model="question.answerTypes.list.placeholder"
+                    data-qa="question-answer-list-placeholder"
```

Comment:
We should have a placeholder prop on this input with it being whatever the default placeholder value is.